### PR TITLE
focus on the last tab.

### PR DESCRIPTION
**Pull request by yqjiang**

### CODE REVIEW

File: elements/js/tabs.js [hunk 1]
```diff
@@ -24,6 +24,7 @@ Drupal.behaviors.xmlFormElementTabs = {
       if(expandedTabs.length > 0) {
         expandedTabs.tabs({
           collapsible: true,
+          selected: expandedTabs.tabs('length')-1,
```
Comment (jordandukart) [inline elements/js/tabs.js]: Is awesome but I think we need to change this to be `expandedTabs.tabs.length`.

If you `console.log(expandedTabs.tabs('length'))` we get back the entire `xml-form-elements-tabs` object. When we subtract 1 from this we result with a NaN (not a number). Seems very odd that it was still working but if we change it to the proposed everything seems to work.

Comment (jordandukart) [inline elements/js/tabs.js]: Though hm, tried this with more than two tabs and doesn't update....

Comment (jordandukart) [inline elements/js/tabs.js]: Ya hm going to need to look at this some more I think.

Comment (yqjiang) [inline elements/js/tabs.js]: Do you know which version of jQuery UI currently use?

File: elements/js/tabs.js [hunk 2]
```diff
@@ -24,6 +24,7 @@ Drupal.behaviors.xmlFormElementTabs = {
       if(expandedTabs.length > 0) {
         expandedTabs.tabs({
           collapsible: true,
+          selected: expandedTabs.children('div').size() - 1,
```
Comment (yqjiang) [inline elements/js/tabs.js]: expandedTabs 
-ul 
--li
--li
-div
-div
get the number of div.
The API of jQuery ui 1.8 says length returns integer.(http://api.jqueryui.com/1.8/tabs/#method-length) but some times not. so use jQuery stuff.

Comment (jordandukart) [inline elements/js/tabs.js]: Unfortunately this seems to work on forms with a single set of tab panels. When there are two or more tabpanels on a page this does not work (newspaper solution pack for example).

File: elements/js/tabs.js [hunk 3]
```diff
@@ -135,8 +145,9 @@ Drupal.behaviors.xmlFormElementTabs = {
     }
   },
   attach: function (context, settings) {
-	  this.tabs.loadPanels(true);
+	  this.tabs.loadPanels(true,context);
```
Comment (adam-vessey) [inline elements/js/tabs.js]: Should have a space after the comma, before `context`.

File: elements/js/tabs.js [hunk 4]
```diff
@@ -31,6 +32,15 @@ Drupal.behaviors.xmlFormElementTabs = {
       if(this.nonCollapsibleTabs.length > 0) {
         this.nonCollapsibleTabs.tabs({});
       }
+      
+      this.tabs.each(function(){
+        if(context.attr('class') === 'clear-block') {
+          var tab = context.find('.xml-form-elements-tabs');
+          tab.tabs({
+            selected: tab.tabs('length')-1
```
Comment (adam-vessey) [inline elements/js/tabs.js]: Spacing before/after the minus sign.

File: elements/js/tabs.js [hunk 5]
```diff
@@ -4,7 +4,7 @@ Drupal.behaviors.xmlFormElementTabs = {
     tabs: null, // Collection of all tabpanels.
     collapsibleTabs: null,
     nonCollapsibleTabs: null,
-    loadPanels: function (collapse) {
+    loadPanels: function (collapse,context) {
```
Comment (adam-vessey) [inline elements/js/tabs.js]: Spacing after the comma, before `context`.

**Pull request merged**